libs/Helpers.py

```diff
@@ -23,3 +24,37 @@ def gte(self, version, target):
         except ValueError:
             # Returning False on exception as a workaround for when an null (or invalid) semver version is passed
             return False
+
+    @keyword
+    def install_rhoam_addon(self, cluster_name):
+        ocm_client = OpenshiftClusterManager()
+        ocm_client.cluster_name = cluster_name
+        ocm_client.install_rhoam_addon(exit_on_failure=False)
+
+    @keyword
+    def uninstall_rhoam_addon(self, cluster_name):
+        ocm_client = OpenshiftClusterManager()
+        ocm_client.cluster_name = cluster_name
+        ocm_client.uninstall_rhoam_addon(exit_on_failure=False)
+
+    @keyword
+    def get_cluster_name(self, cluster_identifier):
+        ocm_client = OpenshiftClusterManager()
+        # to manipulate ocm_describe on line 45
+        ocm_client.cluster_name = cluster_identifier
+        cluster_name = ocm_client.ocm_describe(filter="--json | jq -r '.name'")
+        cluster_name = cluster_name.strip("\n")
+        return cluster_name
+
+    @keyword
+    def is_rhods_addon_installed(self, cluster_name):
+        ocm_client = OpenshiftClusterManager()
+        ocm_client.cluster_name = cluster_name
+        install_flag= ocm_client.is_addon_installed(addon_name="managed-odh")
+        return install_flag
+
+    @keyword
+    def uninstall_rhods_using_addon(self, cluster_name):
```

Comment:
Maybe better `uninstall_rhods_addon` so it's consistent with  `uninstall_rhoam_addon`

Reply:
So maybe the solution will be to change the other one: `uninstall_rhoam_using_addon`. Not a big deal (do what is best for you)

Reply:
done